core/dbt/clients/registry.py

```diff
@@ -80,7 +83,8 @@ def package(name, registry_base_url=None):
 
 
 def package_version(name, version, registry_base_url=None):
-    return _get_with_retries("api/v1/{}/{}.json".format(name, version), registry_base_url)
+    response = package(name, registry_base_url)
+    return response["versions"][version]
```

Comment:
Does the API return all the available versions when we don't pass a version to it? If the version doesn't exist (is that possible?), do we handle that correctly?